cli/kv_command.go

```diff
@@ -463,6 +465,17 @@ func (c *kvCommand) addAction(_ *fisk.ParseContext) error {
 		}
 	}
 
+	if len(c.sources) != 0 {
+		var sources []*nats.StreamSource
+
+		for _, source := range c.sources {
+			sources = append(sources, &nats.StreamSource{
+				Name: source,
```

Comment:
Would this not require people to enter KV_X as the source name in --source?

Reply:
No you don't have to and it's ok if you do, I made the nats.go code handle both cases and use the right stream name either way: https://github.com/nats-io/nats.go/blob/7971018b6c2f0f8b1da88336d415e47940c49ede/kv.go#L418